### ISSUE TITLE
Nitric Oxide 0.08 % Paths for MODY

### ISSUE THREAD

**Issue by sstemann** [NCATSTranslator/Feedback#785]
I'm not super sure how to read these paths, ARAX GUI can't display the ARAGORN results set

https://ui.test.transltr.io/main/results?l=Maturity-onset%20Diabetes%20Of%20The%20Young&i=MONDO:0018911&t=0&r=0&q=f8de3997-f479-49c7-b9ed-b15dc9db9ae4

First path
1. Pancreas has part Exubera?

Last paths are many body parts have part Exubera - are we sure?

<img width="848" alt="image" src="https://github.com/NCATSTranslator/Feedback/assets/38321826/c2bdfbf1-d4c6-40d4-9a0b-dbdaf9b8e66a">

<img width="839" alt="image" src="https://github.com/NCATSTranslator/Feedback/assets/38321826/62a93d6c-12ca-434e-b288-1f6cab3ec42b">

**Comment by cbizon** [comment]
Apparently Exubera is some kind of inhaled insulin.   So the treats MODY part makes sense at least, although it's not clear to me where the treats edge is actually coming from, b/c the Knowledge Source just says "Service Provider". But it could be e.g. drug central or something and wouldn't be surprising.

All the has-part exubera stuff I think should be read as has-part insulin.   Looking at the papers, I don't see why semmed has them labeld as exubera rather than insulin.  @andrewsu @saramsey Do either of you have any insight into that?

The paths of the form X part of (big thing) has part (Exubera) are not very strong.  We have code to filter them out (https://github.com/ranking-agent/aragorn/issues/241) but have not yet tested it.  I suspect that this will remove all the partof/haspart paths.  But the others will remain.

**Comment by Genomewide** [comment]
@sstemann (and all other testers) If you want to when you post these, if you use the share button that is associated with a result when the page opens, it will take you to that specific result and open it.  We did not put this in for testers, but it is a good feature for that!

**Comment by andrewsu** [comment]
> All the has-part exubera stuff I think should be read as has-part insulin. Looking at the papers, I don't see why semmed has them labeld as exubera rather than insulin. @andrewsu @saramsey Do either of you have any insight into that?

I looked at the [first three references](https://pubmed.ncbi.nlm.nih.gov/?term=28713640%2C28654313%2C27813053%5Buid%5D) for the `pancreas - has part - Exubera` edge as reported in the UI.  In all three cases, our semmeddb API returns an `insulin - part of - pancreas` edge using `UMLS:C0021641` for insulin.
* [https://biothings.transltr.io/semmeddb/query?q=predication.pmid:28713640 AND object.umls:C0030274](https://biothings.transltr.io/semmeddb/query?q=predication.pmid:28713640%20AND%20object.umls:C0030274)
* [https://biothings.transltr.io/semmeddb/query?q=predication.pmid:28654313 AND object.umls:C0030274](https://biothings.transltr.io/semmeddb/query?q=predication.pmid:28654313%20AND%20object.umls:C0030274)
* [https://biothings.transltr.io/semmeddb/query?q=predication.pmid:27813053 AND object.umls:C0030274](https://biothings.transltr.io/semmeddb/query?q=predication.pmid:27813053%20AND%20object.umls:C0030274)

I thought this might be an overactive nodenorm issue, but that doesn't seem to be the case:
* https://nodenorm.ci.transltr.io/get_normalized_nodes?curie=UMLS:C0021641&drug_chemical_conflate=true&conflate=true

bottom line, I'm not sure what's going on with that edge, but I'm reasonably confident our ingestion of semmeddb is not the source here...

**Comment by cbizon** [comment]
@saramsey is this coming from KG2's semmeddb?

**Comment by sierra-moxon** [comment]
from relay:
Exubera is an inhaled insulin, but not used this way in the paper.

**Comment by sierra-moxon** [comment]
from relay:
need to retest, might be ok with new NodeNorm. 
also need to figure out where its from.

**Comment by amykglen** [comment]
FYI I'm not finding a `(pancreas)--[has_part]--(Exubera)` link in KG2 (neither dev nor production), and I confirmed that our synonymization doesn't merge Exubera with Insulin (they are two separate nodes). I do see the `(pancreas)--[has_part]--(Insulin)` relationship from SemMedDB in KG2, which uses the Insulin identifier UMLS:C0021641 (same as Andrew reported).

**Comment by sstemann** [comment]
dosage is removed, it doesnt read exubera anymore so i guess its resolved altogether
